traffic_ops/app/lib/UI/Topology.pm

```diff
@@ -848,6 +848,9 @@ sub stringify_cs_ds {
     foreach my $ds ( sort keys %{$csds} ) {
         if ( ref( $csds->{$ds} ) eq 'ARRAY' ) {
             foreach my $map ( @{ $csds->{$ds} } ) {
+                if ( ! defined($map) ) {
```

Comment:
This changes the diff text to ignore `undef` in the hash from the JSON array. For reasons I can't figure out, Perl deserializes these JSON keys from HTTP with extra undefs in the hash, but not when loading from the database. 

Both were generated by the same Go function, and as far as I can tell the text is identical. There are no nulls or anything else in the JSON array text, whitespace is the same. I have no idea why one Perl hash ends up with undefs when decoding, and the other doesn't. But this fixes the diff to ignore them.